<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>87</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>207</width>
     <height>84</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="main_layout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="auto_2">
        <property name="text">
         <string>Auto Indent</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="font_button">
        <property name="text">
         <string>Font's settings</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Tab Width</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="tab_width">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>12</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="tabs">
            <property name="text">
             <string>Tabs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="spaces">
            <property name="text">
             <string>Spaces</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Default language</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="langs"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
